stdlib/universe/map_vectorize_float_test.flux

```diff
@@ -91,3 +92,38 @@ testcase vec_with_float_const {
 
     testing.diff(want, got)
 }
+
+testcase vec_with_float_typed_null {
+    expect.planner(rules: ["vectorizeMapRule": 1])
+
+    // When the input to float() is a typed null, the output will be a null of
+    // type float since this matches the column type generally. The input type
+    // is not propagated - essentially we do a type cast of other-null to float-null.
+    want =
+        array.from(
+            rows: [{x: 1, _null: debug.null(type: "int"), output: debug.null(type: "float")}],
+        )
+
+    got =
+        array.from(rows: [{x: 1, _null: debug.null(type: "int")}])
+            |> map(fn: (r) => ({r with output: float(v: r._null)}))
+
+    testing.diff(want, got)
+}
+
+testcase vec_with_float_untyped_null {
+    expect.planner(rules: ["vectorizeMapRule": 1])
+
+    // When the input to float() is an untyped null, the output is also an
+    // untyped null meaning the column is dropped from the output.
+    want =
+        array.from(rows: [{x: 1}])
+            |> debug.opaque()
+
+    got =
+        array.from(rows: [{x: 1}])
+            |> debug.opaque()
+            |> map(fn: (r) => ({r with output: float(v: r._null)}))
+
+    testing.diff(want, got)
```

Comment:
You could use `assertEqualValues` instead so you do not have to construct the streams manually just to compare one value.

Reply:
Yeah, it seems like if I wanted to use `assertEqualValues` for this case, I'd need to also leverage `tableFind` et al. I think I'd rather leave it as-is.

Largely the point of these two testcases is to test the null interaction of `_vecFloat` which is only possible via `map()` right now.